server/selinux/server/pulp-celery.te

```diff
@@ -59,6 +60,8 @@ allow celery_t pulp_var_run_t:file manage_file_perms;
 allow celery_t pulp_var_run_t:dir manage_dir_perms;
 files_pid_filetrans(celery_t, pulp_var_run_t, file)
 
+# needed for ostree
+allow celery_t gpg_exec_t:file { read execute open execute_no_trans };
```

Comment:
Can this be rewritten and moved into the lower section. After asking in #selinux it was suggested to use:

```
gpg_exec(celery_t) ;
```

Also can it be moved into the [lower section](https://github.com/beav/pulp/blob/ostree-gpg-selinux/server/selinux/server/pulp-celery.te#L98-L112) with a comment to the right of the statement.